### PR TITLE
Add config-api example to outputVerificationIgnoreList

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$title

### CODE REVIEW

File: examples/config-api/tests/config_api_test.bal [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
-    } else {
-        test:assertFail(msg = "Failed to call the endpoint:");
-    }
-}
-
-function stopService() {
-    //test:stopServices("config-api");
-}
\ No newline at end of file
+//import ballerina/test;
```
Comment (niveathika) [inline examples/config-api/tests/config_api_test.bal:1]: Any reason to comment the test out?

Comment (aashikam) [reply examples/config-api/tests/config_api_test.bal:1]: needs the command ```ballerina run example.bal --b7a.config.file=/path/to/conf/file/custom-config-file-name.conf``` to run the tests. There is a discussion on this in the group.

Comment (BuddhiWathsala) [reply examples/config-api/tests/config_api_test.bal:1]: @aashikam can't we do something like this https://github.com/ballerina-platform/ballerina-distribution/blob/d1d1f0d29c468a25d425be65deb25e379a9b4e09/ballerina/build.gradle#L1292

Comment (niveathika) [reply examples/config-api/tests/config_api_test.bal:1]: If it cant.. lets use @test:Config{enable:false}

Comment (aashikam) [reply examples/config-api/tests/config_api_test.bal]: The test file was recently edited to not to read it from the conf file. It should be file now.

**Review by niveathika — APPROVED** [review]

**Review by lafernando — APPROVED** [review]

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**